extensions/wikia/PhalanxII/services/PhalanxService.class.php

```diff
@@ -309,4 +307,12 @@ private function sendToPhalanxDaemon( $action, $parameters ) {
 		}
 		return $res;
 	}
+
+	private function getPhalanxUrl( $action ) {
+		global $wgConsulUrl, $wgConsulServiceTag;
+		
+		$baseurl = ( new ConsulUrlProvider( $wgConsulUrl, $wgConsulServiceTag ) )->getUrl( 'phalanx' );
+		return sprintf( "http://%s/%s", $baseurl, $action != "status" ? $action : "" );
+	}
+
 };
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1116)